apps/src/codebridge/InfoPanel/styles/validated-instructions.module.scss

```diff
@@ -44,23 +45,13 @@
   pointer-events: auto;
   flex: 1;
   opacity: 1;
+  height: 100%;
 
   &.itemVertical {
     flex-direction: column;
   }
 }
 
-.itemTitle {
```

Comment:
this was not being used, so I cleaned it up.

Reply:
🧹 🧹 🧹